res/translations/mixxx_nl.ts

```diff
@@ -6187,12 +6215,12 @@ U kunt de Tracks ten allen tijde op het scherm slepen en neerzetten om een deck
     <message>
         <location filename="../../src/preferences/dialog/dlgprefeffectsdlg.ui" line="317"/>
         <source>❯</source>
-        <translation type="unfinished"/>
+        <translation>❯</translation>
```

Comment:
Should this really be a translateable string?

Reply:
This is unrelated to this PR. But I actually don't mind. If you think that should be changed, file a GitHub issue.